python/convert_to_dataset.py

```diff
@@ -0,0 +1,128 @@
+"""Convert Synapse Folder to Dataset
+
+TODO:
+  - use Python client once Dataset support becomes available
+  - get Folder annotations and add them to Dataset entity
+  - apiKey will be deprecated, update to PAT in login()
+
+author: verena.chung
+"""
+
+import os
+import json
+import getpass
+
+import synapseclient
+from synapseclient import Column
+
+PARENT_ID = "syn21498902"
+FOLDERS_ID = "syn22047105"
+
+
+def login():
+    """Log into Synapse. If env variables not found, prompt user.
+
+    Returns:
+        syn: Synapse object
+    """
+    try:
+        syn = synapseclient.login(
+            os.getenv('SYN_USERNAME'),
+            apiKey=os.getenv('SYN_APIKEY'),
```

Comment:
Don't use API key - use synapse PAT

Reply:
I know, it's already been noted in my TODO on line 6.

Reply:
On this note, the [docs](https://python-docs.synapse.org/build/html/index.html#synapseclient.Synapse.login) should be updated to reflect this change, e.g. I didn't realize PAT is already accepted.

Reply:
`authToken` is the PAT

Reply:
Oh, d'oh. In retrospect, that makes sense.

Thanks!  I'll implement the update.